src/coffee/stockimport.coffee

```diff
@@ -104,11 +108,15 @@ class StockImport
       Promise.reject "#{CONS.LOG_PREFIX}Unknown import mode '#{mode}'!"
 
   summaryReport: (filename) ->
-    if @_summary.created is 0 and @_summary.updated is 0
+    if @_summary.created is 0 and @_summary.updated is 0 and (not @shouldRemoveZeroInventories or @_summary.removed is 0)
       message = 'Summary: nothing to do, everything is fine'
     else
+      deletedStocksMessage = ""
+      if @shouldRemoveZeroInventories
+        deletedStocksMessage = " and #{@_summary.removed} were deletions"
+
       message = "Summary: there were #{@_summary.created + @_summary.updated} imported stocks " +
-        "(#{@_summary.created} were new and #{@_summary.updated} were updates)"
+        "(#{@_summary.created} were new, #{@_summary.updated} were updates#{deletedStocksMessage})"
```

Comment:
could we maybe move the space ` ` into this string instead of having it inside the `deletedStocksMessage` variable? Then it would align better me thinks.

Reply:
But when this feature is turned off it would print:
`(123 were new, 456 were updates )` <- the space before the closing bracket.

Reply:
Ahh, true!